CircleciScripts/prepare_xcframework_archives.py

```diff
@@ -2,9 +2,11 @@
 import sys
 import shutil
 import re
+import logging
```

Comment:
Thanks, I was thinking about replacing the custom log message with the logging module. However, by doing this, you will lose the timestamp on messages. Also, don't we have to set the log level somewhere to ensure that our `info` level messages are actually output?